packages/php-wasm/compile/build.js

```diff
@@ -167,7 +167,10 @@ if (!requestedVersion || requestedVersion === 'undefined') {
 const sourceDir = path.dirname(new URL(import.meta.url).pathname);
 
 // Build the base image
-await asyncSpawn('make', ['base-image'], { cwd: sourceDir, stdio: 'inherit' });
+await asyncSpawn('make', [], {
```

Comment:
it would be nice here to leave some explicit intention in the code. although it _happens_ to work with the changes you made to the `Makefile`, we've lost the linkage between asking for what we want (with the pre-existing `base-image`) and running `make` and hoping it does what we need (because no more target is listed).

adding `base-image` here would be helpful, or `all`, or anything to communicate to our future selves what our purupose was in running `make`.